LeafOutlook/FormSettings.cs

```diff
@@ -255,6 +265,21 @@ private void loadStartWithAudioMutedButtons()
             }
         }
 
+        private void loadRibbonOnEveryAppointmentButtons()
+        {
+            if (Properties.Settings.Default.showRibbonOnEveryAppointment)
```

Comment:
You can just direct assign and use a temp var if you feel it's too long. like:

```
bool showRibbon =Properties.Settings.Default.showRibbonOnEveryAppointment;
radioButtonRibbonOnEveryAppointmentToggled.Checked = showRibbon;
radioButtonRibbonOnEveryAppointmentUntoggled.Checked = !showRibbon;
```

probably a lame var name but you get the idea.

Reply:
Makes total sense, also probably an attempt to be consistent with the rest of the code, but changed it a you asked because less code